use-shopping-cart/utilities/old-utils.test.js

```diff
@@ -1,4 +1,8 @@
-import { formatCurrencyString, getCheckoutData, filterCart } from './util'
+import {
+  formatCurrencyString,
+  getCheckoutData,
+  filterCart
+} from '../core/store'
```

Comment:
@dayhaysoos I'm fixing up a bunch of things that seem to be broken rn. When we were discussing importing and exporting certain values, I think there was a miscommunication. The values still live in `utilities/old-utils`.

Also, I don't have a good name for that file yet. Couldn't think of one, maybe if we split up the values into separate files like (`formatCurrencyString` could be in `utilities/formatting.js` and `getCheckoutData` and `filterCart` could be in `utilities/cart.js`). I'll probably actually change those names real quick unless you have some ideas for the filenames.